workspaces/ui-v2/src/pages/changelog/ChangelogEndpointRootPage.tsx

```diff
@@ -213,28 +213,111 @@ const ChangelogRootComponent: FC<
         )}
 
         {thisEndpoint.requestBodies.map((requestBody) => (
-          <TwoColumnBodyChangelog
-            changesSinceBatchCommitId={batchId}
-            rootShapeId={requestBody.rootShapeId}
-            bodyId={requestBody.requestId}
-            location={'Request Body'}
-            contentType={requestBody.contentType}
-            description={requestBody.description}
-          />
+          <div
+            className={classes.bodyContainer}
+            id={requestBody.requestId}
+            key={requestBody.requestId}
+          >
+            <div className={classes.bodyHeaderContainer}>
+              <h6 className={classes.bodyHeader}>Request Body</h6>
+              <ReactMarkdown
+                className={classes.contents}
+                source={requestBody.description}
+              />
+            </div>
+            <div className={classes.bodyDetails}>
+              <div>
+                <ContributionFetcher
```

Comment:
The biggest callout here is that we use `ContributionFetcher` and `ShapeFetcher` which _both_ call `useShapeDescriptor` on the same root shape - meaning there's double the spectacle calls. This _shouldn't_ be an issue since spectacle is either in memory or hitting a local cli network. And this should be temporary since I'm planning to move this into redux (the idea here is that if these are split, it's much easier to incrementally split these out)